### PR TITLE
Fix game_restart()

### PR DESCRIPTION
**Pull request by Toad06**
It's currently broken because `Run_EndGame()` calls `Audio_Quit()` which destroys the AudioContext instance. This ends up causing a crash.

### CODE REVIEW

File: scripts/_GameMaker.js [hunk 1]
```diff
@@ -2306,10 +2312,10 @@ function GameMaker_Tick()
 
 		        case ROOM_ENDOFGAME:
                 case ROOM_ABORTGAME:
-                    Run_EndGame();
+                    Run_EndGame(false);
                     return;
```
Comment (JohnyDaison) [inline scripts/_GameMaker.js]: Since you're touching this part of code, could you please align this case statement with the others?

Comment (Toad06) [reply scripts/_GameMaker.js]: Done - I applied the changes to the whole switch statement.

**Comment by toby-yoyo** [comment]
Linked this to [GM-6297](https://bugs.opera.com/browse/GM-6297).

**Review by toby-yoyo — APPROVED** [review]
I think this is good to go in - thank you for your contribution!

**Pull request merged**